lang/en/tool_excimer.php

```diff
@@ -107,6 +109,7 @@
 $string['scripttype_cli'] = 'CLI';
 $string['scripttype_ajax'] = 'Ajax';
 $string['scripttype_ws'] = 'Service';
+$string['scripttype_cron'] = 'Cron';
```

Comment:
From a naming point of view its slightly more Moodly to talk about Tasks and the Task API vs cron (I'm pretty sloppy with this myself)